issue session cookie doesn't expire

OmarTariq612 opened the pull request:
Not calling the function makes the created cookie a session cookie instead of making it last for a month.

![image](https://github.com/zjunior-infra/zJunior/assets/65053572/a9346ce3-21c0-4a83-b83a-5aaaac042dbb)

![Screenshot from 2023-05-31 01-04-55](https://github.com/zjunior-infra/zJunior/assets/65053572/a0830ff3-42a2-4a50-8763-d0833de09c3d)

![image](https://github.com/zjunior-infra/zJunior/assets/65053572/a87b3bc0-fe81-4a70-b60d-1540128b8695)

Mahmoudgalalz reviewed (changes requested):
thanks for this @OmarTariq612 I just want to add an additional change to that, to avoid making this uncompiled in the client's code

removing the directive `is:inline` will show errors in the IDE as it will think that the DOM API isn't available it's a known issue in Astro within TS configs, and needs an update to allowJs If you could do that too will be a great addition, 

so for merging it it should be complete the needs as will
- remove is:inline
- add allowJs in ts config

OmarTariq612 commented:
I removed the `is:inline` directive but I didn't have errors calling the DOM API, the only errors I faced are related to types so I fixed them, also I noticed that `cookie` and `url` constants aren't being used so I removed them.

Mahmoudgalalz commented on src/pages/newjob/index.astro:
This is awesome @OmarTariq612 Thanks for the non-noticeable bug

Mahmoudgalalz commented:
we hope you would like to join us on [Slack](https://join.slack.com/t/juniorjobs/shared_invite/zt-1t3koh4xe-HqiPVA7umjpZ4O8EjEpFzg)

to discuss and be involved in early contribution, before adding them in the repo
